pom.xml

```diff
@@ -28,7 +28,7 @@
         <changelist>-SNAPSHOT</changelist>
         <jenkins.version>2.121.1</jenkins.version>
         <java.level>8</java.level>
-        <workflow-api-plugin.version>2.28</workflow-api-plugin.version>
+        <workflow-api-plugin.version>2.30</workflow-api-plugin.version>
```

Comment:
Updated to fix `RequireUpperBoundDeps` caused for updating to workflow-support-3.0:
```
Require upper bound dependencies error for org.jenkins-ci.plugins.workflow:workflow-api:2.28 paths to dependency are:
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-api:2.28
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-api:2.28
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-job:2.11.2
    +-org.jenkins-ci.plugins.workflow:workflow-api:2.15
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-basic-steps:2.6
    +-org.jenkins-ci.plugins.workflow:workflow-api:2.16
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-cps:2.32
    +-org.jenkins-ci.plugins.workflow:workflow-api:2.15
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-cps:2.32
    +-org.jenkins-ci.plugins.workflow:workflow-api:2.15
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-durable-task-step:2.13
    +-org.jenkins-ci.plugins.workflow:workflow-api:2.19
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-support:3.0
    +-org.jenkins-ci.plugins.workflow:workflow-api:2.30
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins:pipeline-build-step:2.5.1
    +-org.jenkins-ci.plugins.workflow:workflow-api:2.3
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-multibranch:2.10
    +-org.jenkins-ci.plugins.workflow:workflow-api:2.3
```